### PR TITLE
Rdm dev

### PR DESCRIPTION
**Pull request by doerlbh**
Hi guys, I created a vanilla case for RDM class for people who is eager to use. The methods are still pending to be further implemented though, including transforms and dissimilarity computation etc..

### CODE REVIEW

**Comment by doerlbh** [comment]
Added RDMs class and implemented the two main functions:
**get_vectors**()
**get_matrices**()

**Comment by HeikoSchuett** [comment]
I shortly went and fixed the unit-test issues. Please have a look through the code climate report and fix those. You can also run these tests with pylint pyrsa locally. 
Otherwise: Great that we now have something to work on.

File: pyrsa/util/indicator.py
Comment (JasperVanDenBosch) [inline pyrsa/util/indicator.py]: On first look it seems like `identity` and `identity_pos` are quite similar, perhaps  `identity_pos` could call `identity` with an flag `positive=True`? It then checks for this flag and does the `cc[cc>0]` step. This solves the code replication warning.

**Comment by doerlbh** [comment]
Will work on the code climate stuff later this week or early next week. Other than that, the unit tests now are all passing with the latest updates from data-dev.

**Comment by doerlbh** [comment]
@jdiedrichsen @ilogue @HeikoSchuett @iancharest Feel free to checkout the latest version of **rdm-dev**. All tests are passing (unit tests and style tests). It would be great to merge the current stage soon, before more developmental changes are made.

File: pyrsa/data/dataset.py
Comment (JasperVanDenBosch) [inline pyrsa/data/dataset.py]: I think these should go in #16 right?

File: tests/test_data.py
Comment (JasperVanDenBosch) [inline tests/test_data.py]: Is this for the Simulation package? It does not seem to test RDM code?

File: pyrsa/rdm/__init__.py
Comment (JasperVanDenBosch) [inline pyrsa/rdm/__init__.py]: I think this should be `from pyrsa.rdm.rdms import RDMs`

File: pyrsa/util/indicator.py
Comment (JasperVanDenBosch) [inline pyrsa/util/indicator.py]: On first look it seems like `identity` and `identity_pos` are quite similar, perhaps  `identity_pos` could call `identity` with an flag `positive=True`? It then checks for this flag and does the `cc[cc>0]` step. This solves the code replication warning.

File: tests/test_indicator.py
Comment (JasperVanDenBosch) [inline tests/test_indicator.py]: In the future, it may be useful also to test the contents, e.g. by testing one specific cell. Unless only the shapes of these matrixes matter? I am not very familiar with the concept of indicator matrices.

**Review by JasperVanDenBosch — CHANGES_REQUESTED** [review]
Good work on the style fixes, I have a few more requests, in particular which PR will contain the dataset classes and to reduce the complexity of the indicator module.

File: pyrsa/data/dataset.py
Comment (doerlbh) [reply pyrsa/data/dataset.py]: Yes, that's true, but since I accidentally start my branch rdm-dev from the data-dev, so I have to manually update it in rdm-dev from data-dev, before pull #16 is approved and merged.

File: tests/test_data.py
Comment (doerlbh) [reply tests/test_data.py]: I haven't implemented the test part yet for the rdm classes. I plan to do that as a next step (perhaps in the next pull), or do you suggest me to provide the test now in this pull?

File: pyrsa/rdm/__init__.py
Comment (doerlbh) [reply pyrsa/rdm/__init__.py]: fixed and pushed.

File: tests/test_data.py
Comment (doerlbh) [reply tests/test_data.py]: I will add test_rdm.py in my next few pushes. Thanks for the note.

File: pyrsa/util/indicator.py
Comment (doerlbh) [reply pyrsa/util/indicator.py]: Thanks for the note! Will take a look and change in the next few pushes.

File: pyrsa/util/indicator.py
Comment (doerlbh) [reply pyrsa/util/indicator.py]: Thanks for the note! Will take a look and change in the next few pushes.

File: tests/test_indicator.py
Comment (doerlbh) [reply tests/test_indicator.py]: Thanks for the note! Will take a look and change in the next few pushes. I think the shape is important to check to avoid dimension conflicts, while adding content check will definitely be essential as well.

File: pyrsa/data/dataset.py
Comment (JasperVanDenBosch) [reply pyrsa/data/dataset.py]: I'm sorry I don't think I understand. Why's that? Are the dataset classes imported by the rdm modules?

File: tests/test_data.py
Comment (doerlbh) [reply tests/test_data.py]: fixed and pushed.

File: pyrsa/util/indicator.py
Comment (doerlbh) [reply pyrsa/util/indicator.py]: fixed and pushed.

File: pyrsa/util/indicator.py
Comment (doerlbh) [reply pyrsa/util/indicator.py]: fixed and pushed.

File: tests/test_indicator.py
Comment (doerlbh) [reply tests/test_indicator.py]: Fixed and pushed. As suggested, I added a simple content check. For more extensive content checks, I think Joern would be a better person to add them as he first implemented indicator functions and is more familiar to the concept, and might know which cells are more important to add in test cases.

**Comment by doerlbh** [comment]
Implementations and unit tests for the class with two main functions are all now complete and passing all criteria. Helpful remarks and requested changes from @ilogue are all fixed in the current version. Please review the code and let me know if anything else need to be changed, and approve if you think it's ready. Thanks!

**Comment by JasperVanDenBosch** [comment]
@doerlbh The next step on this I think is to merge master (remove the Data contents from this branch)

**Comment by doerlbh** [comment]
@ilogue Thanks! FYI, I have merged all the updates from the master branch, and they are all cleaned up. Feel free to check it out~

**Review by JasperVanDenBosch — APPROVED** [review]
Looks good!

**Review by saarbuckle — APPROVED** [review]
Good stuff

**Pull request merged**
